src/sagemaker/model.py

```diff
@@ -345,6 +375,16 @@ def compile(
                 model. Allowed values: 'mxnet', 'tensorflow', 'keras', 'pytorch',
                 'onnx', 'xgboost'
             framework_version (str):
+            target_platform_os (str): Target Platform OS. Allowed values: 'LINUX', 'ANDROID'.
+                It can be used instead of target_instance_family.
+            target_platform_arch (str): Target Platform Architecture. Allowed values: 'X86_64',
+                'X86', 'ARM64', 'ARM_EABIHF', 'ARM_EABI'.
+                It can be used instead of target_instance_family.
+            target_platform_accelerator (str): Target Platform Accelerator. Allowed values:
+                'INTEL_GRAPHICS', 'MALI', 'NVIDIA'. Optional.
+            compiler_options (str): Additional parameters for compiler (Optional, in JSON format).
```

Comment:
We may need to point out here as well that `target_instance_family` and `target_platform_*` are not usable together?

Reply:
For each `target_platform_*` field documentation I added - `It can be used instead of target_instance_family.`